ui/src/app/components/cases/list.component.html

```diff
@@ -198,7 +198,13 @@
           </a>
         </div>
         <div class="ts-col-15 px-25"
-             [textContent]="'testcase.status_'+testCase?.status| translate"></div>
+             [textContent]="'testcase.status_'+testCase?.status| translate">
+        </div>
+        <div *ngIf="isStepGroup && !isDeletedFilter" class="ts-col-10 px-25">
+          <i
+            class="fa-trash-thin action-icon" (click)="$event.preventDefault();fetchLinkedTestCases(testCase?.id); $event.stopPropagation()"
```

Comment:
create a event handler and move this there

Reply:
done
check latest commit